lib/loader/index.d.ts

```diff
@@ -8,7 +8,7 @@ export interface ResultObject {
 
 /** WebAssembly imports with an optional env object and two levels of nesting. */
 export type Imports = {
-  [key: string]: Record<string,unknown>;
+  [key: string]: Record<string,unknown> | undefined;  /** env?: {} not allowed without the '| undefined' anymore */
```

Comment:
```suggestion
  [key: string]: Record<string,unknown> | undefined;
```
Could you remove this comment? I think it's unnecessary

Reply:
Done.